src/basic/terminal-util.c

```diff
@@ -455,7 +455,7 @@ int acquire_terminal(
                                         goto fail;
                                 }
 
-                                r = fd_wait_for_event(fd, POLLIN, ts + timeout - n);
+                                r = fd_wait_for_event(notify, POLLIN, ts + timeout - n);
```

Comment:
@fbuihuu again, this code is supposed to be compatible with anyone taking the console, not just protect ourselves against ourselves. hence, the lock files wouldn't work, because nobody but us would care. and again, lockfiles are awful, as they never clean themselves up as mentioned.